### PR TITLE
Corrections, additions to initial doc

### PR DESCRIPTION
**Pull request by mattreecebentley**
See individual commits for details, but 3rd option provided appears not to work.
Overall I think the documentation is poor for first-timers, these are merely some minor corrections to reduce some of the confusion.

### CODE REVIEW

**Comment by mattreecebentley** [comment]
I signed

**Comment by dominichamon** [comment]
Thanks so much for helping clean up the documentation for first time users. If you have any other specific issues, please feel free to raise an issue, create a PR, or contact us on freenode IRC at #googlebenchmark.

**Comment by mattreecebentley** [comment]
Thanks - I think it needs a lot more work, it's just that I don't actually understand what's going on enough at this point to re-write it. There's no details about what methodology the benchmarks is using, and I can only pick up anecdotal stuff elsewhere.

**Comment by dominichamon** [comment]
If you have specific questions then I think we'll be able to tackle them,
and then we can use that to add more to the documentation. Feel free to
start a thread on the benchmark-discuss google group so we can capture it.

Dominic Hamon | Google
*There are no bad ideas; only good ideas that go horribly wrong.*

**Comment by mattreecebentley** [comment]
It's more that there's just no detail there. No information about what the individual macro/functions are doing, nothing. I don't feel like this is something that even requires discussion, it's that bizarre, to me.

File: README.md [hunk 1]
```diff
@@ -88,6 +87,11 @@ to use, test, and provide feedback on the new features are encouraged to try
 this branch. However, this branch provides no stability guarantees and reserves
 the right to change and break the API at any time.
 
+##Prerequisite knowledge
```
Comment (LebedevRI) [inline README.md]: While this may be a valid markdown, it isn't displayed correctly on github,
A space is missing.

**Pull request merged**